cmd/commands/common.go

```diff
@@ -360,44 +356,21 @@ func getKubeContextNameFromUserSelect(cmd *cobra.Command, kubeContextName *strin
 	return nil
 }
 
-func getIngressHostFromUserInput(cmd *cobra.Command, ingressHost *string) error {
-	if ingressHost != nil && *ingressHost != "" {
-		*ingressHost = strings.TrimSpace(*ingressHost)
-		return nil
-	}
-
-	if store.Get().Silent {
-		return fmt.Errorf("missing ingress host")
-	}
-
-	ingressHostPrompt := promptui.Prompt{
-		Label: "Ingress host (required)",
-	}
-
-	ingressHostInput, err := ingressHostPrompt.Run()
+func getIngressHostFromCluster(ctx context.Context, opts *RuntimeInstallOptions) error {
+	fmt.Print("Retrieving ingress controller info from your cluster...\n")
+	cs := opts.KubeFactory.KubernetesClientSetOrDie()
+	ingressController, err := cs.CoreV1().Services(opts.IngressNamespace).Get(ctx, opts.IngressController, metav1.GetOptions{})
 	if err != nil {
-		return fmt.Errorf("Prompt error: %w", err)
-	}
-
-	ingressHostInput = strings.TrimSpace(ingressHostInput)
-	if ingressHostInput == "" {
-		return fmt.Errorf("missing ingress host")
+		return fmt.Errorf("failed to get ingress controller info from your cluster: %w", err)
 	}
 
-	die(cmd.Flags().Set("ingress-host", ingressHostInput))
-	*ingressHost = ingressHostInput
+	opts.IngressHost = fmt.Sprintf("https://%s", ingressController.Status.LoadBalancer.Ingress[0].Hostname)
```

Comment:
do we want to maybe check which ports the LoadBalancer is listening on, to decide if we want to use https or http?